baselines/dom.generated.d.ts

```diff
@@ -14627,6 +14649,178 @@ declare var HTMLSummaryElement: {
     new(): HTMLSummaryElement;
 };
 
+interface EXT_blend_minmax {
+    readonly MIN_EXT: number;
+    readonly MAX_EXT: number;
+}
+
+interface EXT_frag_depth {
+}
+
+declare var EXT_frag_depth: {
```

Comment:
you do not need all the `declare var`s. please remove them all.

Reply:
Fixed also.